internal/providers/terraform/azure/testdata/dns_a_record_test/dns_a_record_test.golden

```diff
@@ -0,0 +1,20 @@
+
+ Name                                    Monthly Qty  Unit                  Monthly Cost 
+                                                                                         
+ azurerm_dns_a_record.first1B                                                            
+ └─ DNS queries (first 1B)                     1,000  1M queries                 $400.00 
+                                                                                         
+ azurerm_dns_a_record.over1B                                                             
+ ├─ DNS queries (over 1B)                      9,000  1M queries               $1,800.00 
+ └─ DNS queries (first 1B)                     1,000  1M queries                 $400.00 
```

Comment:
Would be nice if we show users first tier as first value to keep the price order :)